src/debug_commands.c

```diff
@@ -1324,6 +1353,8 @@ DebugCommandType commands[] = {{"DUMP_INVIDX", DumpInvertedIndex}, // Print all
                                {"DUMP_HNSW", dumpHNSWData},
                                {"SET_MONITOR_EXPIRATION", setMonitorExpiration},
                                {"WORKERS", WorkerThreadsSwitch},
+                               {"INDEXES", ListIndexesSwitch},
```

Comment:
Consider `_LIST` or `LIST`, so it resembles its un-obfuscated counterpart.
```suggestion
                               {"_LIST", ListIndexesSwitch},
```

Reply:
Didn't we talk about wanting to deprecate `_LIST`?
Are we sure we want to follow the same name?

Reply:
I think while they have the same semantics they should have the same name. At the end changing this name is not breaking so we can do it any day.
Is there a new name on the line for the original command we can already use here?

Reply:
The problem is that if we have scripts that interact with an azure cluster it will complicate their logic if for one cluster then need to use LIST and in another some other command.

@DvirDukhan do you have a preference on the subcommand name?